js/scripts.js

```diff
@@ -1,21 +1,20 @@
 (function(){
 	const flipButton = document.getElementById("flip");
-	let coin = ["",""]
 	const messageBox = document.getElementById("messageBox");
 	const tails = document.getElementById("tails");
 	const heads = document.getElementById("heads");
 	const coinSpin = document.getElementById("spin");
 
 	tails.onclick = function() {
-		flipACoin();
-		checkGuess("tails", coin[0]);
-		animation(coin);
+		let headsOrTails = flipACoin();
```

Comment:
We want to actually get this result and then pass it through to the next 2 functions instead of relying on a global state.